tt_eager/tt_dnn/op_library/complex/complex_ops.hpp

```diff
@@ -72,18 +72,11 @@ class ComplexTensor {
         }
 };
 
-Tensor is_real(const ComplexTensor& input, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
-Tensor is_imag(const ComplexTensor& input, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
-Tensor real(const ComplexTensor& input, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
-Tensor imag(const ComplexTensor& input, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
-
 ComplexTensor conj(const ComplexTensor& input, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
-Tensor angle(const ComplexTensor& input, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
 
 ComplexTensor complex_add(const ComplexTensor& input_a, const ComplexTensor& input_b, const MemoryConfig& output_mem_config);
```

Comment:
Any reason not to just use `add` instead of `complex_<op>` ?

Reply:
We are overloading the ops such that both nornal tensor type and complex tensor type are handled by ttnn.add itself. Only in the function definition we are using names like complex_add

Reply:
I expect that C++ is also properly overloaded. C++ API is a first class citizen

Reply:
Hi @eyonland , @ayerofieiev-tt ,
We have overloaded complex_abs with unary abs, such that when abs is called, based on the input type, it gets mapped. Add will be done in today's task.